src/VisualStudio/TestUtilities2/ProjectSystemShim/Framework/TestEnvironment.vb

```diff
@@ -179,6 +183,18 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim.Fr
                     Case GetType(SVsFileChangeEx)
                         Return _fileChangeEx
 
+                    Case GetType(SVsLinkedUndoTransactionManager)
```

Comment:
What is demanding these now?

Reply:
This change is now removed from this pull request.